fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -196,11 +196,17 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	// Given an array that's ascend sorted by CPU percent, the pivot position is CPU_PIVOT_RATIO *
 	// team count. DD won't move shard to teams that has CPU > pivot CPU.
 	double CPU_PIVOT_RATIO;
+	// TODO@muzhi: Whether use the same knos as `AVAILABLE_SPACE_PIVOT_RATIO`.
+	double DD_STRICT_AVAILABLE_SPACE_PIVOT_RATIO;
+	// TODO@muzhi: Whether use the same knos as `CPU_PIVOT_RATIO`.
+	double DD_STRICT_CPU_PIVOT_RATIO;
```

Comment:
Did you mean to add these TODOs? I think it is good to keep these knobs separate for now to allow for tuning as the feature rolls out, for example, if we need to make this feature less aggressive without affecting other things. It does add testing burden, but that seems worth it to control this feature in prod. In the future, we can assign `DD_STRICT_CPU_PIVOT_RATIO = CPU_PIVOT_RATIO` by default (and similarly for space pivot) and phase out the new knobs if needed.